examples/lambda/docker/lambda/index.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+
+from momento import CacheClient, Configurations, CredentialProvider
+from momento.responses import CacheGet, CacheSet, CreateCache
+
+
+def handler(event, lambda_context):
+    cache_name = "default-cache"
+    with CacheClient(
+            configuration=Configurations.Laptop.v1(),
+            credential_provider=CredentialProvider.from_environment_variable("MOMENTO_AUTH_TOKEN"),
+            default_ttl=timedelta(seconds=60),
+    ) as cache_client:
```

Comment:
so we will change this to use your new stuff once we have a release?

Reply:
correct